.changeset/metal-bottles-happen.md

```diff
@@ -0,0 +1,36 @@
+---
+'@finos/babel-preset-legend-studio': patch
+'@finos/eslint-plugin-legend-studio': patch
+'@finos/legend-application': patch
```

Comment:
You don't need this file. We only add changesets to the `packages` we make changes too. Please delete this file and add the desciption to the other changeset you added

Reply:
Added a new single changeset for the three packages